extensions/amp-analytics/0.1/events.js

```diff
@@ -22,6 +22,30 @@ import {user} from '../../../src/log';
 const VARIABLE_DATA_ATTRIBUTE_KEY = /^vars(.+)/;
 const NO_UNLISTEN = function() {};
 
+/** @enum {string} */
+const WAITFOR_EVENTS_LIST = {
+  NONE: 'none',
+  INI_LOAD: 'ini-load',
+  RENDER_START: 'render-start',
+};
+
+/**
+ * @interface
+ */
+class SignalTrackerDef {
+  /**
+   * @param {?string} unusedEventType
+   * @return {!Promise}
+   */
+  getRootSignal(unusedEventType) {}
+
+  /**
+   * @param {!Element} unusedElement
```

Comment:
If we make `eventType` `{string=}` I can't switch them. I am good with either way.